### PR TITLE
Much, much cleanup. Requires review by another contributor to be merged.

### PR DESCRIPTION
**Pull request by Eladkay**
The code is, in theory, a little less awful now. I'd like another review on this to make sure I hadn't broken anything.

### CODE REVIEW

File: src/main/java/eladkay/scanner/terrain/TileEntityTerrainScanner.java [hunk 1]
```diff
@@ -257,10 +237,9 @@ private boolean checkForBlock(IBlockState local, BlockPos imm) { //True if the b
         if (block.isReplaceable(getWorld(), imm) || block.isAir(local, getWorld(), imm)) { //Replaceable / air
             if (!(block instanceof BlockFluidBase) && !(block instanceof BlockLiquid)) { //Not occupied by liquid
                 return true;
-            }
-            else if (Config.replaceNonSourceLiquid) { //Occupied by liquid, check for config, if config is on then replace non-source blocks
-                if (local.getValue(BlockFluidBase.LEVEL) < 15) //Non-source
-                    return true;
+            } else if (Config.replaceNonSourceLiquid) { //Occupied by liquid, check for config, if config is on then replace non-source blocks
+                //Non-source
+                return local.getValue(BlockFluidBase.LEVEL) < 15;
```
Comment (misterplus) [inline src/main/java/eladkay/scanner/terrain/TileEntityTerrainScanner.java]: Maybe do 
```
return (local.getValue(BlockFluidBase.LEVEL) < 15 || local.getValue(BlockLiquid.LEVEL) < 15);
```
Just to make sure vanilla liquid works.

**Review by misterplus — APPROVED** [review]
Looks good, I may add additional changes for behavior changes, but that's for another pull request.

File: src/main/java/eladkay/scanner/terrain/TileEntityTerrainScanner.java (hunk 1 above)
Comment (Eladkay) [reply src/main/java/eladkay/scanner/terrain/TileEntityTerrainScanner.java]: Fixed in 94527b5.

**Review by misterplus — APPROVED** [review]
Looks good!

**Review by misterplus — APPROVED** [review]
Looks alright, I'll add zh_cn.lang in a future commit.

**Pull request merged**